server.go

```diff
@@ -194,13 +194,22 @@ func (b *backend) PropPatch(r *http.Request, update *internal.PropertyUpdate) (*
 }
 
 func (b *backend) Put(w http.ResponseWriter, r *http.Request) error {
-	err := b.FileSystem.Create(r.Context(), r.URL.Path, r.Body)
+	fi, err := b.FileSystem.Create(r.Context(), r.URL.Path, r.Body)
 	if err != nil {
 		return err
 	}
 
+	w.Header().Set("Content-Length", strconv.FormatInt(fi.Size, 10))
+	if fi.MIMEType != "" {
+		w.Header().Set("Content-Type", fi.MIMEType)
+	}
+	if !fi.ModTime.IsZero() {
+		w.Header().Set("Last-Modified", fi.ModTime.UTC().Format(http.TimeFormat))
+	}
+	if fi.ETag != "" {
+		w.Header().Set("ETag", internal.ETag(fi.ETag).String())
+	}
 	w.WriteHeader(http.StatusCreated)
-	// TODO: Last-Modified, ETag, Content-Type if the request has been copied
 	// verbatim
```

Comment:
Sorry, but I'm going to bother you again: "verbatim" is part of the sentence but isn't been deleted :P

Reply:
:see_no_evil: brain-off coding is not a good thing ...